src/assets/scripts/airport/Arrival/ArrivalBase.js

```diff
@@ -1,56 +1,182 @@
-import $ from 'jquery';
-import _has from 'lodash/has';
+import _get from 'lodash/get';
+import _map from 'lodash/map';
 import _random from 'lodash/random';
-
+import FixCollection from '../Fix/FixCollection';
+import RouteModel from '../Route/RouteModel';
 import PositionModel from '../../base/PositionModel';
+import {
+    airlineNameAndFleetHelper,
+    randomAirlineSelectionHelper
+} from '../../airline/randomAirlineSelectionHelper';
 import { nm, degreesToRadians } from '../../utilities/unitConverters';
 import { round, sin, cos } from '../../math/core';
-import { distance2d } from '../../math/distance';
-import { bearing, fixRadialDist, inAirspace, dist_to_boundary } from '../../math/flightMath';
-import { vradial, vsub } from '../../math/vector';
+import {
+    bearing,
+    fixRadialDist,
+    isWithinAirspace,
+    calculateDistanceToBoundary,
+    calculateHeadingFromTwoPositions
+} from '../../math/flightMath';
+import { FLIGHT_CATEGORY } from '../../aircraft/AircraftInstanceModel';
+import { AIRPORT_CONSTANTS } from '../../constants/airportConstants';
+import { TIME } from '../../constants/globalConstants';
 import { LOG } from '../../constants/logLevel';
 
 /**
- *  Generate arrivals at random, averaging the specified arrival rate
+ * @property INTERVAL_DELAY_IN_MS
+ * @type {number}
+ * @final
+ */
+const INTERVAL_DELAY_IN_MS = TIME.ONE_HOUR_IN_SECONDS;
+
+// TODO: this shouldn't live here. perhaps move to `FixCollection` as an exported function?
+/**
+ * Encapsulation of a `FixCollection` method.
+ *
+ * This allows for centralization of this logic, while avoiding the need for
+ * another class method.
+ *
+ * @method getFixPostiion
+ * @param fixName {string}
+ * @return fix.position {array}
+ */
+const getFixPosition = (fixName) => {
+    const fix = FixCollection.findFixByName(fixName);
+
+    return fix.position;
+};
+
+/**
+ * Generate arrivals at random, averaging the specified arrival rate
  *
  * @class ArrivalBase
  */
 export default class ArrivalBase {
```

Comment:
Not sure if it's been on the horizon or what, but the way all this was set up seemed strange to me, with these different classes, and each one inheriting `ArrivalBase` and then applying their own corrected methods. Since you are the simplification guy, I'm curious about your thoughts on how this could be arranged better, perhaps as separate methods of a singular class, or whatever the case may be. Again, little detail needed, just a concept I'm curious about.